inductiva/api/methods.py

```diff
@@ -70,13 +79,17 @@ def upload_input(api_instance: TasksApi, task_id, original_params,
         original_params: Params of the request passed by the user.
         type_annotations: Annotations of the params' types.
     """
+    logging.info("Creating input archive...")
     input_zip_path = pack_input(
         params=original_params,
         type_annotations=type_annotations,
         zip_name=task_id,
     )
+    file_size = os.path.getsize(input_zip_path)
+    logging.info("Input archive created.")
+    logging.info("Input archive size: %s", utils.format_bytes(file_size))
```

Comment:
Do you want to add this to the downloads or you will do it in another way?

Reply:
It will have to be done differently, so I'll do it later in a different PR.